File: OnlineDeliveryServer/OnlineDeliveryServer/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OnlineDeliveryServer
{
    public class Product
    {
        public string Name { get; set; }
        public int Id { get; set; }
        public string Ingredients { get; set; }
        public double Price { get; set; }
        public Product() { }

        public Product(string name, int id, string ingredients, double price)
        {
            Name = name;
            Id = id;
            Ingredients = ingredients;
            Price = price;
        }
    }
}

File: WebProject/src/model/Product.ts
export class Product
{
    id: number = 0;
  name: string = '';
  ingredients: string = '';
  price: number = 0;

  public ToString(product: Product) {
    return (
      'ID: ' +
      product.id +
      ' Name: ' +
      product.name +
      ' Price: ' +
      product.price
    );
  }
}